website/src/components/EventsList.tsx

```diff
@@ -1,29 +1,42 @@
 import * as React from 'react'
-import EventData from "../data/events-listing"
+import EventData from '../data/events-listing'
 
 class Conferences extends React.Component {
-	render() {		
+	render() {
 		return (
-			<div class="card-deck">
-				
+			<div className="card-deck">
 				{EventData.Conference.map((eventDetail, index)=> {
-				 return ( 					
-					<div class="card col-sm-3">
-					    <img class="card-img-top img-fluid" src={eventDetail.eventImage} />
-					    <div class="card-body">
-					    	<h5>{eventDetail.eventTitle}</h5>
-					    	<p>{eventDetail.eventLocation}<br />
-					    	{eventDetail.eventDate}<br />
-					    	<a href={eventDetail.eventLink} ref="nofollow">Join us &rarr;</a></p>
-					    </div>
-					</div>
-				)
-
+					return ( 					
+						<div className="card col-sm-3 event__item">
```

Comment:
```suggestion
						<div className="card col-sm-4 event__item">
```

Reply:
Since we are only listing 3 events currently, this change will center the events instead of positioning them too far left.

With change:
![image](https://user-images.githubusercontent.com/9110008/76255288-2ef21200-620b-11ea-82e1-7709d1e6e890.png)

Vs:
![image](https://user-images.githubusercontent.com/9110008/76255320-3b766a80-620b-11ea-9012-0f69de28e1b6.png)